test/unit/std/charconv_test.cpp

```diff
@@ -270,16 +280,18 @@ TYPED_TEST(from_char_real_test, real_numbers)
         TypeParam val{};
         std::string str = "1234";
         auto res = std::from_chars(&str[0], &str[0] + str.size(), val);
-        EXPECT_EQ(val, TypeParam{1234});
+        EXPECT_TRUE(double_equals(val, TypeParam{1234}));
```

Comment:
cool thanks! I didn't know they have this

Reply:
.... My tests fail with `EXPECT_DOUBLE_EQ` from google test since the epsilon is too small :scream: 
```
  val // from_chars output
    Which is: -0.30000001192092896
  TypeParam{-0.3} // direct initialization
    Which is: -0.29999999999999999
```

... @h-2 @joergi-w Do you think this is too severe? ... But I think if anything this is the fault of `std::strtof`

Reply:
Yes, that's exactly what I would expect :)

Reply:
alright float then